source/messaging/channel-preferences.rst

```diff
@@ -18,53 +18,95 @@ Set Channel Preferences
   :target: https://mattermost.com/deploy
   :alt: Available for Mattermost Self-Hosted deployments.
 
-You can set channel preferences, such as notifications, as well as the channel's header, purpose, and name for each channel you belong to. To access these settings, select the channel name at the top of the page.
+For each channel you belong to, you can set notification preferences as well as the channel's header, purpose, and name. Select the channel name at the top of the screen to access channel-specific settings.
+
+.. image:: ../images/channel-preferences.png
+        :alt: Select the channel name at the top of the screen to set preferences for each channel you belong to.
 
 Channel notification preferences
 --------------------------------
 
-Each channel you belong to can have different notification preferences.
+Change the following channel notification settings by selecting **Notification Preferences**.
 
 Mute channel
 ~~~~~~~~~~~~~
 
-By default, channel muting is turned off for all channels. To mute or unmute a channel, select the channel name at the top of the page to access the channel menu, then choose **Mute Channel**.
+By default, all channels are unmuted. This means that desktop, email, and push notifications display for all channels, and when you're mentioned, mention badges display in the Channel Sidebar. To mute the current channel and disable all desktop, email, and push notifications, enable the **Mute Channel** option, then select **Save**. You will continue to see mention badges when you mute a channel.
```

Comment:
This paragraph is a bit hard to read. Perhaps:

"Muting turns off desktop, email and push notifications for a channel, and it will only be marked as unread in the channel sidebar if you're mentioned. To mute the current channel and disable notifications select **Mute Channel** from the channel name at the top of the screen. By default, all channels are unmuted."